.gitignore

```diff
@@ -1,8 +1,14 @@
 .jujuignore
 .vscode/
+.build/
+.tox/
+.venv/
 build/
 env/
 venv/
 src/__pycache__/
-tests/__pycache__/
-apt-mirror.charm
+tests/unit/__pycache__/
```

Comment:
I think that we can use `__pycache__/` to ignore any `__pycache__` folder.

Reply:
Sorry it should be `**/__pycache__`

Reply:
Done